plugins/woocommerce-admin/client/customize-store/assembler-hub/index.tsx

```diff
@@ -183,7 +183,7 @@ export const AssemblerHub: CustomizeStoreComponent = ( props ) => {
 				<ShortcutProvider style={ { height: '100%' } }>
 					<OptInContextProvider>
 						<GlobalStylesProvider>
-							<RouterProvider>
+							<RouterProvider routes={ [] } pathArg="p">
```

Comment:
I think that we can remove `pathArg`:

```suggestion
							<RouterProvider routes={ [] }>
```

Reply:
Oh true, I just merged it.